omnigibson/object_states/saturated.py

```diff
@@ -14,7 +14,7 @@ def _get_value(self, system):
         return self.obj.states[ParticleRemover].check_at_limit(system=system)
 
     def _set_value(self, system, new_value):
-        assert_valid_key(key=system, valid_keys=ParticleRemover.supported_systems, name="particle system")
+        assert_valid_key(key=system, valid_keys=ParticleRemover.supported_active_systems, name="particle system")
```

Comment:
why does the system need to be active? couldn't it be initialized dynamically during the get() call?

Reply:
I am being a bit more conservative here. Essentially when you call `obj.states[Saturated].set_value(get_system("XYZ"))`, `get_system` itself already initializes the system.